### DISCUSSION TITLE
Do not know how to load path: pocketbase

### DISCUSSION THREAD

**Discussion by richardltc** [pocketbase/pocketbase#183] [Q&A] [answered]
Hi, I am trying to use Pocketbase with SvelteKit and am getting the error: `Do not know how to load path: pocketbase` using the example JavaScrypt SDK when trying to create a new user:

import PocketBase from 'pocketbase';

const client = new PocketBase('http://localhost:8090');

Any ideas?

Richard

**Comment by ganigeorgiev** [comment, marked as answer]
Hi, this error is usually thrown when the module you are loading is not installed yet.

Could you confirm that there is a `pocketbase` entry in your package.json `dependencies/devDependencies` section? (if not run `npm install pocketbase --save`).

If that doesn't work, could you check what version of the JavaScript SDK you are using?

**Reply by richardltc** [reply]
Thanks for the fast reply:

```
	"devDependencies": {
		"@sveltejs/adapter-auto": "next",
		"@sveltejs/kit": "next",
		"eslint": "^8.16.0",
		"eslint-plugin-svelte3": "^4.0.0",
		"pocketbase": "^0.2.1",
		"svelte": "^3.44.0",
		"svelte-check": "^2.7.1",
		"typescript": "^4.7.4",
		"vite": "^2.9.13"
	},
	"type": "module",
	"dependencies": {
		"pocketbase": "^0.2.1"
	}
```

**Reply by ganigeorgiev** [reply]
You don't need to have the `pocketbase` module registered in both `devDependencies` and `dependencies` sections.

I'll try to reproduce the issue locally with a new SvelteKit project and will get back to you.

**Reply by ganigeorgiev** [reply]
@richardltc I'm not able to reproduce the issue. I've just tested it with a new SvelteKit project and everything seems to load fine. These are the commands that I've executed:
```sh
npm create svelte@latest my-app
cd my-app
npm install
npm install pocketbase --save
npm run dev
```
And I've loaded the SDK by editing `src/routes/index.svelte`:
```html
<script>
	import Counter from "$lib/Counter.svelte";
	import PocketBase from "pocketbase";

	const client = new PocketBase("http://localhost:8090");

	async function loadMessages() {
		const list = await client.Records.getList("messages");
		console.log(list);
	}

	loadMessages();
</script>
```

You could try deleting your `node_modules` and run again `npm install`.

**Reply by richardltc** [reply]
Strange I had to re-create the proect again, but that fixed it. Thank you :)
